modules/safer_mysql/variables.tf

```diff
@@ -0,0 +1,338 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "project_id" {
+  description = "The project ID to manage the Cloud SQL resources"
+}
+
+variable "name" {
+  description = "The name of the Cloud SQL resources"
+}
+
+// required
+variable "database_version" {
+  description = "The database version to use"
+}
+
+// required
+variable "region" {
+  description = "The region of the Cloud SQL resources"
+  default     = "us-central1"
+}
+
+// required
+variable "vpc_network" {
+  description = "Existing VPC network to which instances are connected. The networks needs to be configured with https://cloud.google.com/vpc/docs/configure-private-services-access."
+}
+
+variable "vpc_peering" {
```

Comment:
Could this be incorporated into the module itself, or possibly as a submodule?

Reply:
I defined it as a different submodule, and updated the example and tests to use it.

it can't be incorporated in the module itself, as it is created only once for all Cloud SQL instances associated to the same VPC.

Reply:
ah sorry, I overlooked this comment